### ISSUE TITLE
[SF][A11y][UploadCollection]: Cannot set the group label for the UploadCollection

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents#3768]
**Describe the bug**
The uploadCollection is a listbox, but there is no option to set the group aria-label. 

**Isolated Example**
*Please provide an isolated example if possible by forking [this codesandbox](https://codesandbox.io/s/ui5-webcomponents-react-template-29l34).*

**To Reproduce**

**Expected behavior**
Screen reader should announce the grouping label of listbox as it will be descriptive for the user to understand the group label and the number of attachments

**Screenshots**
![image](https://user-images.githubusercontent.com/1614528/131458146-5bd4ed48-3e76-44de-acbc-c9338ada7b3d.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:0.31.15
`@ui5/webcomponents-react` version:0.131.15
Operating System:macOS Big Sur 11.5
Browser:Google Chrome Version 92.0.4515.159 (Official Build) (x86_64)

**Additional context**

**Comment by fifoosid** [comment]
FYI @SAP/ui5-webcomponents-topic-rd

**Comment by TeodorTaushanov** [comment]
Hi @ericchan1983,

In the master branch we added "accessibleName" property to the upload collection, which you need to set to the group title.

In your case it can be like:

```
	<ui5-upload-collection accessible-name="Attachment (1)">
		<div slot="header">
			<ui5-title>Attachment (1)</ui5-title>
```


Best,
Teo

**Comment by ericchan1983** [comment]
Hi @TeodorTaushanov,

I am from SuccessFactors Web Experience Shanghai team.

Our team is only allowed to use the SF version of ui5-webcomponent.
We can not use rc version of ui5-webcomponent.

If this issue is fixed in master branch. Can you please downport this feature to 0.31.x ? ( The current version is 0.31.17, maybe you can release 0.31.18 )

Regards,
Eric Chen

**Comment by ilhan007** [comment]
Hi @ericchan1983 we will downport it.

**Comment by ericchan1983** [comment]
Hi @Lukas742,

Once the feature is downported, can you help to upgrade the ui5-wc-react to support the new property on react component?

Regards,
Eric Chen

**Comment by ilhan007** [comment]
Hi @ericchan1983 first we need to release a new patch, then we will  notify ui5-wc-react (as we always do)
to release new patch on their side. We wait for few more issues to be fixed before releasing, probably we will do that on Monday.

**Comment by Lukas742** [comment]
Hi @ericchan1983 

depending on the effort on our side, we usually release no later than 3 days after the sf release of the web components, in most cases it's on the same or next day.

**Comment by ilhan007** [comment]
Fixed with 0.31.18 - new accessibleName property has been introduced for the purpose.

**Comment by ilhan007** [comment]
Hi @ericchan1983 

I downported the **accessibleName**, **accessibleNameRef**, **accessibleRole**  properties that the List already have in the master to the 0.31 branch with this [commit](https://github.com/SAP/ui5-webcomponents/commit/e84a3fa84a32d30eb3b59632f2433f9fcf65e5fb).

Prior to this change the List did not have **accessibleName** property in 0.31 and this is the reason for the issue. 
It will be resolved with the next patch.

**Comment by ericchan1983** [comment]
Hi @ilhan007,

Great, thanks for the quick response.

Regards,
Eric Chen
